### PR TITLE
GPII-4186: Using windows certificate store for certificates.

### PR DESCRIPTION
**Pull request by stegru**
Fixes a problem where the Office ribbon wasn't being downloaded (`SELF_SIGNED_CERT_IN_CHAIN`).

This was due to it being run behind an invasive https proxy, and node using a hard-coded list of CA certificates, rather than those provided by the OS.

This fix instructs node to use the Windows certificate store, which means certificates that the administrator has added are also trusted. In other words, if it works in the browser it will work with node.

Unsure why https connections to the gpii cloud still worked, however.

*To reproduce:*
* Get the root certificate for `untrusted-root.badssl.com`, and add it to the *Trusted Root Certifications Authorities* certificate store. (remove it after)
* In [win32.json](https://github.com/GPII/universal/blob/master/testData/solutions/win32.json5#L16132-L16132) Replace `raw.githubusercontent.com` with `untrusted-root.badssl.com` and key-in as `otis`
* or set GPII_CLOUD_URL to `https://untrusted-root.badssl.com`

There will be an error of `SELF_SIGNED_CERT_IN_CHAIN` in the log. After applying this fix, there will be a `404` error (meaning https is working).

### CODE REVIEW

**Comment by javihernandez** [comment]
> CI job failed: https://ci.gpii.net/job/windows-tests/1057/

FYI, this failure was caused by the update of the windows vagrant box to 1903 on CI. We are performing tests right now and I will trigger this again when we revert it back to 1803. Sorry for any inconvenience.

File: package.json [hunk 1]
```diff
@@ -23,7 +23,8 @@
         "ref-struct": "1.1.0",
         "ref-array": "1.1.2",
         "string-argv": "0.0.2",
-        "rimraf": "2.6.2"
+        "rimraf": "2.6.2",
+        "win-ca": "^3.1.0"
```
Comment (javihernandez) [inline package.json]: Can you please fix the version number?

**Comment by javihernandez** [comment]
@stegru I just tested this and worked great. In addition to the request above (https://github.com/GPII/windows/pull/281#discussion_r337079593), can you also update the PR with current master?

**Comment by stegru** [comment]
ok to test

**Comment by stegru** [comment]
ok to test

**Pull request merged**